### PR TITLE
Switch Pulp to use sqlite3 as default

### PR DESCRIPTION
**Pull request by bmbouter**
https://pulp.plan.io/issues/3359
closes #3359

### CODE REVIEW

**Comment by daviddavis** [comment]
The Travis tests are failing because they run the migrations to test them out. Any chance you can update the Travis file to run the migrations against both sqlite and PostgreSQL? It would be a good check to ensure compatibility with both databases.

File: pulpcore/pulpcore/app/settings.py [hunk 1]
```diff
@@ -168,9 +168,9 @@
     # https://docs.djangoproject.com/en/1.8/ref/settings/#databases
     'databases': {
         'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': 'pulp',
-            'USER': 'pulp',
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': '',
```
Comment (daviddavis) [inline pulpcore/pulpcore/app/settings.py]: Looks like you still need a db name here.

Comment (dkliban) [reply pulpcore/pulpcore/app/settings.py]: yeah ... thename should be 'pulp'

**Comment by bmbouter** [comment]
I pushed those changes, but it says it's running Ruby for some reason on Travis? [This Job](https://travis-ci.org/pulp/pulp/jobs/347519979) has config:

```


{
  "language": "ruby",
  "os": "linux",
  "group": "stable",
  "dist": "trusty"
}
```

hmmmm?

**Comment by bmbouter** [comment]
ok test

**Comment by daviddavis** [comment]
The .travis.yml had bad escaping in it. I fixed it here:

https://github.com/pulp/pulp/pull/3348

File: pulpcore/pulpcore/etc/pulp/server.yaml [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
+# consider configuring Postgresql or another Django supported database.
 #
 # databases:
 #   default:
 #     CONN_MAX_AGE: 0
-#     ENGINE: django.db.backends.postgresql_psycopg2
-#     NAME: pulp
-#     USER: pulp
+#     ENGINE: django.db.backends.sqlite3
+#     NAME:
```
Comment (daviddavis) [inline pulpcore/pulpcore/etc/pulp/server.yaml]: fyi, this needs to be 'pulp'

**Comment by bmbouter** [comment]
Closing because daviddavis fixed it! https://github.com/pulp/pulp/pull/3348#pullrequestreview-100279278

**Pull request closed without merging**